content/tokio/tutorial/channels.md

```diff
@@ -2,8 +2,29 @@
 title: "Channels"
 ---
 
-Now that we have learned some about concurrency with Tokio, let's apply this on
-the client side. Say we want to run two concurrent Redis commands. We can spawn
+Now that we have learned a bit about concurrency with Tokio, let's apply this on
+the client side. Put the server code we wrote before into an explicit binary 
+file:
+
+    mkdir src/bin
+    mv src/main.rs src/bin/server.rs
```

Comment:
You will need to change this to a fenced code block for our CI setup to not consider this as Rust code that it can run.

    ```text
    mkdir src/bin
    mv src/main.rs src/bin/server.rs
    ```